build/package/Dockerfile.python-toolset

```diff
@@ -10,6 +13,13 @@ ENV PYTHONUNBUFFERED=1 \
 
 RUN microdnf install -y python38 tar
 
+RUN if [ ! -z ${nexusHost} ] && [ ! -z ${nexusAuth} ]; \
+    then pip3 config set global.index-url https://${nexusAuth}@${nexusHost}/repository/pypi-all/simple \

```

Comment:
TBH, I'm not a fan of this approach. This bakes the credentials of one shared user into the image, which is something which typically should not be done I believe.

I think I'd prefer to supplying these to the running container via secrets. That way it'll be easier to migrate to per-project credentials later on I think. Thoughts?

Reply:
agree, then we need to test if we can setup pip on the fly